cos/static/stats.js

```diff
@@ -293,15 +293,15 @@ function display_road_and_army(){
 //Call to redraw tables with information from the backend
 function update_tables(data,stage,layer){
     var players = data.Players;
-        for (i = 0; i < 3; i++) {
-            var elm = document.getElementById("player_name" + i);
-            elm.innerText = players[i].Player.player_name;
-            var color = players[i].Player.player_color;
-            if (color == 'white') {
-                elm.style.color = 'black';
-            }
-            else {
-                elm.style.color = color;
-            }
+    for (i = 0; i < 3; i++) {
+        var elm = document.getElementById("player_name" + i);
+        elm.innerText = players[i].Player.player_name;
+        var color = players[i].Player.player_color;
+        if (color == 'white') {
+            elm.style.color = 'black';
         }
+        else {
+            elm.style.color = color;
+        }
+    }
```

Comment:
The only change I'm noticing here is getting rid of some whitespace. It's a good change if that's what it is, I just want to make sure I'm not misreading it.

Reply:
Yes, that is right. I edited the file when doing some testing to debug a problem and cleaned up the formatted when done.